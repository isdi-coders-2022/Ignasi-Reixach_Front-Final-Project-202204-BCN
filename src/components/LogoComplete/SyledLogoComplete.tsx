import styled from "styled-components";

export const StyledLogoComplete = styled.div`
  background-color: #c6c9bf;
  display: flex;
  flex-direction: column;
  align-content: center;
  align-items: center;
  padding: 20px;
  margin-bottom: -10px;

  .container-logo {
    width: 60%;
    display: flex;
    flex-direction: row;
    margin-top: 0%;
    align-items: flex-end;
    justify-content: center;

    h1 {
      margin: 0;
      padding: 0;
      color: #1c3b5e;
      font-size: 60px;
      letter-spacing: 5px;
    }

    img {
      display: flex;
      width: 100px;
      height: 100px;
      margin-left: 20px;
      transform: translateY(-4px);
    }
  }

  p {
    color: #4242429e;
    text-align: center;
    font-weight: 900;
    letter-spacing: 2px;
    margin-top: 10px;
    font-size: 16px;
    margin-bottom: 40px;
  }
`;
