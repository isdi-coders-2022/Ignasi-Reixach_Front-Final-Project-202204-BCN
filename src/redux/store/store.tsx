import { configureStore } from "@reduxjs/toolkit";
import dishesReducer from "../features/dishesSlice";

import userReducer from "../features/userSlice";

const store = configureStore({
  reducer: {
    user: userReducer,
    dishes: dishesReducer,
  },
});

export type RootState = ReturnType<typeof store.getState>;
export type AppDispatch = typeof store.dispatch;
export default store;
