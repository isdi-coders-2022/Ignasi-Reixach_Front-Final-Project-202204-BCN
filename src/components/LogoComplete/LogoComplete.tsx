import { StyledLogoComplete } from "./SyledLogoComplete";

const LogoComplete = () => {
  return (
    <>
      <StyledLogoComplete>
        <div className="container-logo">
          <h1>
            Dish<br></br>Week
          </h1>
          <img src="/images/icons/bowl-food.svg" alt="Icon bowl of food" />
        </div>
        <p>
          Hello there, create your <br></br>weekly menu
        </p>
      </StyledLogoComplete>
    </>
  );
};

export default LogoComplete;
