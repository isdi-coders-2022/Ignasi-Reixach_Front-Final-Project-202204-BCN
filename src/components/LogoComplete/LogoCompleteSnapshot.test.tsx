import { Provider } from "react-redux";
import { BrowserRouter } from "react-router-dom";
import TestRenderer from "react-test-renderer";
import store from "../../redux/store/store";
import LogoComplete from "./LogoComplete";

describe("Given a LogoComplete", () => {
  describe("When it's rendered", () => {
    test("Then it should match this snapshot", () => {
      const LogoCompleteTest = TestRenderer.create(
        <BrowserRouter>
          <Provider store={store}>
            <LogoComplete />
          </Provider>
        </BrowserRouter>
      );

      expect(LogoCompleteTest).toMatchSnapshot();
    });
  });
});
