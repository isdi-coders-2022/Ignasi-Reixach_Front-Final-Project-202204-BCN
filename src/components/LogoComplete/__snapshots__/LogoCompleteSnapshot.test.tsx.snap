// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Given a LogoComplete When it's rendered Then it should match this snapshot 1`] = `
<div
  className="sc-bczRLJ dLJPrl"
>
  <div
    className="container-logo"
  >
    <h1>
      Dish
      <br />
      Week
    </h1>
    <img
      alt="Icon bowl of food"
      src="/images/icons/bowl-food.svg"
    />
  </div>
  <p>
    Hello there, create your 
    <br />
    weekly menu
  </p>
</div>
`;
